api/envoy/config/accesslog/v2/wasm.proto

```diff
@@ -16,13 +16,5 @@ import "envoy/config/wasm/v2/wasm.proto";
 // that calls into a WASM VM. Configures the built-in *envoy.wasm_access_log*
 // AccessLog.
 message WasmAccessLog {
-  // A unique ID for a set of filters/services in a VM which will share Context(s).
-  string root_id = 1;
-  // A unique ID for a VM so that multiple filters/services can call into the same VM.
-  string vm_id = 2;
-  //  Wasm configuration string e.g. a serialized protobuf which will be the argument to the
-  // VM configure() call.
-  string configuration = 3;
-  // Configuration for starting a new VM to be associated with the given vm_id.
-  envoy.config.wasm.v2.VmConfig vm_config = 4;
+  envoy.config.wasm.v2.PluginConfig config = 1;
```

Comment:
Note: Envoy uses `extensions` (vs `modules` in Apache/IIS/lighttpd/LiteSpeed/NGINX/Varnish and `plugins` in ATS), so I'm not sure if we should be using `plugin` here... but let's leave this discussion for upstreaming.

Reply:
Sounds good to me.